numpy/core/src/multiarray/dtypemeta.h

```diff
@@ -7,10 +7,9 @@ extern "C" {
 
 #include "numpy/_dtype_api.h"
 
-/* DType flags, currently private, since we may just expose functions */
+/* DType flags, currently private, since we may just expose functions 
+   Other publicly visible flags are in _dtype_api.h                   */
 #define NPY_DT_LEGACY 1 << 0
-#define NPY_DT_ABSTRACT 1 << 1
-#define NPY_DT_PARAMETRIC 1 << 2
```

Comment:
Deleted these two since they already appear in `_dtype_api.h`, I left `NPY_DT_LEGACY` since it's a private flag.